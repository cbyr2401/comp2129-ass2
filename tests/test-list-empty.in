LIST ENTRIES
LIST KEYS
LIST SNAPSHOTS
BYE