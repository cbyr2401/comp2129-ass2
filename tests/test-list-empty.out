> no entries

> no keys

> no snapshots

> bye